forch/faucetizer.py

```diff
@@ -208,12 +208,9 @@ def _faucetize(self):
                 if self._has_acl(acl_name):
                     port_cfg['acls_in'] = [acl_name]
                 else:
-                    port_cfg['acls_in'] = []
                     LOGGER.error('No ACL defined for role %s', device_behavior.role)
-            else:
-                port_cfg['acls_in'] = []
             if self._config.tail_acl:
-                port_cfg['acls_in'].append(self._config.tail_acl)
+                port_cfg.setdefault('acls_in', []).append(self._config.tail_acl)
```

Comment:
I think maybe your title is misleading?  Seems like this will configure the acls_in with the tail_acl even if the device does not have a role?

Reply:
Updated